tests/integration/suite/sentry/utils.go

```diff
@@ -78,3 +104,38 @@ func signJWT(builder *jwt.Builder) ([]byte, error) {
 
 	return jwt.Sign(token, jwt.WithKey(jwa.ES256, jwtSigningKeyPriv))
 }
+
+func validateCertificateResponse(t *testing.T, res *sentrypbv1.SignCertificateResponse, sentryBundle ca.CABundle, expectDNSName string) {
+	t.Helper()
+
+	require.NotEmpty(t, res.WorkloadCertificate)
+
+	rest := res.WorkloadCertificate
+
+	// First block should contain the issued workload certificate
+	{
+		var block *pem.Block
+		block, rest = pem.Decode(rest)
+		require.NotEmpty(t, block)
+		require.Equal(t, "CERTIFICATE", block.Type)
+
+		cert, err := x509.ParseCertificate(block.Bytes)
+		require.NoError(t, err)
+
+		assert.Equal(t, []string{expectDNSName}, cert.DNSNames)
```

Comment:
Also check the SPIFFE ID is correct for the app in the signed certificate. This is contained in the `cert.URIs` field.

Reply:
Please also check that the certificate has the `server auth` and `client auth` key usages.

Reply:
Done